### PR TITLE
Update translation strings

### PR DESCRIPTION
**Pull request by nlehuby**
* Remove line numbers and fuzzy matching from the the translations strings collections
* Update `po` files with the checks recently added

@frodrigo can you please review the translations files, upload them to transifex and tell me if you want me to work on some automatic update for transifex (see #25 )?

### CODE REVIEW

**Comment by frodrigo** [comment]
Do you fetch from transifex before ?

File: po/fa.po [hunk 1]
```diff
@@ -1,122 +1,142 @@
...
-#: ../transport.validator.mapcss:5
 msgid "Uncompromising validation of transit data"
 msgstr ""
 
-#: ../transport.validator.mapcss:39 ../transport.validator.mapcss:44
 msgid "Missing transportation mode, add a tag route = bus/coach/tram/etc"
-msgstr ""
-"شیوهٔ حمل‌ونقل عمومی مشخص نشده. یکی از تگ‌های route = bus/coach/tram/...‎ را"
-" وارد کنید."
+msgstr "    .    route = bus/coach/tram/...   ."
```
Comment (frodrigo) [inline po/fa.po]: Translation lost

File: po/cs.po [hunk 2]
```diff
@@ -2,118 +2,146 @@
...
 
-#: ../transport.validator.mapcss:50
 msgid "Missing transportation mode, change tag route to route_master"
 msgstr ""
 
-#: ../transport.validator.mapcss:66
 msgid ""
 "Missing public_transport:version tag on a public_transport route relation"
-msgstr "Chybějící tag public_transport:version na relaci public_transport"
+msgstr "Chybjc tag public_transport:version na relaci public_transport"
```
Comment (frodrigo) [inline po/cs.po]: Look like unicode issue.

**Comment by frodrigo** [comment]
I'm not sure about removing the lines number is a good practice.

**Comment by nlehuby** [comment]
You are right, I forgot to pull the translations first! This is now done.

File: po/fa.po (hunk 1 above)
Comment (nlehuby) [reply po/fa.po]: the encoding was not set in the po, I don't know why :thinking: I've added it to fix this. Thanks for noticing !

File: po/cs.po (hunk 2 above)
Comment (nlehuby) [reply po/cs.po]: same here, fixed by adding the encoding

**Comment by frodrigo** [comment]
Now it os ok for me.

**Comment by nlehuby** [comment]
About the line numbers: I don't know what they are useful for, and they produce ugly diffs as the line numbers keep changing on each commit.
But I can put them back if you prefer.

**Comment by frodrigo** [comment]
let's try to remove

**Comment by nlehuby** [comment]
If it's ok for you @frodrigo can you please upload the new po to transifex (I don't have the permissions to do so) and maybe to Osmose too ?

**Comment by frodrigo** [comment]
I was waiting for this to be merged

**Comment by frodrigo** [comment]
Push to transifex done.

**Comment by frodrigo** [comment]
Note, on fetch again from transifex line numbers are back.

**Comment by nlehuby** [comment]
And the encoding notices are removed again ... This is quite annoying!

How do you update the po files ?

**Comment by frodrigo** [comment]
I do not remember, probably with the make file. I think charset in the files are not required.

**Comment by nlehuby** [comment]
I don't think so: the `msgmerge` failed if it's not defined. It was the issue you noticed with the fa.po and cs.po file.

And how do you upload them to transifex ? With the CLI or via the website ?

**Comment by frodrigo** [comment]
- maybe linked to some var env.
- "tx" cli command, installed using python virtualenv.

**Comment by nlehuby** [comment]
pff, why is i18n always such a pain in the ass 🤯

**Pull request merged**